docs/en/cloud/AWS-EMR.md

```diff
@@ -31,93 +31,192 @@ Blob Store.
-Args=[s3://test-bucket/path/to/mount/] \
---configurations https://alluxio-public.s3.amazonaws.com/emr/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-emr.json \
---ec2-attributes KeyName=admin-key
+sudo runuser -l alluxio -c "/opt/alluxio/bin/alluxio runTests"
 ```
 
-> Note: The default Alluxio Worker memory is set to 1/3 of the physical memory on the instance.
-If a specific value is desired, set `alluxio.worker.memory.size` in the provided
-`alluxio-site.properties` or in the additional options argument.
+  {% endcollapsible %}
+{% endaccordion %}
+
+{% accordion setup details %}
+  {% collapsible Cluster details %}
+Using this boostrap script, Alluxio is installed in `/opt/alluxio/` by default.
+Hive and Presto are already configured to connect to Alluxio.
+The cluster also uses AWS Glue as the default metastore for both Presto and Hive.
+This will allow you to maintain table definitions between multiple runs of the Alluxio cluster.
+
+The default Alluxio worker is allocated a third of the instance's available memory.
+If the instance type has less than 20GB of memory, you will need to make your own copy of the `alluxio-emr.sh` script
+and update the divisor value at `mem_div=3`.
+
+  {% endcollapsible %}
+{% endaccordion %}
 
 ## Creating a Table
 
 The simplest step to using EMR with Alluxio is to create a table on Alluxio and query it using Presto/Hive.
 
-1. SSH into the 'hadoop' user in the master node.
-2. Create a directory in Alluxio to be the external location of your table.
+{% accordion table ssh %}
+  {% collapsible SSH into the master node %}
+From your terminal, SSH into the master instance using the key pair provided in the `create-cluster` command.
+```console
+ssh -i /path/to/keypair.pem hadoop@masterPublicDns
+```
+Note that we are connecting as the `hadoop` user.
+All subsequent commands assume they are being executed from within the instance.
+
+  {% endcollapsible %}
+{% endaccordion %}
+
+{% accordion table create directory %}
+  {% collapsible Create a directory in Alluxio to be the external location of your table %}
+Create the `/testTable` directory in Alluxio, then set the `hadoop` user to be the directory owner.
+Note that these commands are being executed as the `alluxio` user.
 ```console
-$ /opt/alluxio/bin/alluxio fs mkdir /testTable
+sudo runuser -l alluxio -c "/opt/alluxio/bin/alluxio fs mkdir /testTable"
+sudo runuser -l alluxio -c "/opt/alluxio/bin/alluxio fs chown hadoop:hadoop /testTable"
 ```
-3. Start the hive CLI.
+
+  {% endcollapsible %}
+{% endaccordion %}
+
+{% accordion table hive %}
+  {% collapsible Create a new database in AWS GLUE %}
+Open the Hive CLI.
 ```console
-$ hive
+hive
 ```
-4. Create a new database to see if AWS Glue is working as expected.
-Check the [console](https://console.aws.amazon.com/glue/home) to see if the database is created.
+
+Create a database, then check in the [GLUE console](https://console.aws.amazon.com/glue/home) to see if the database is created.
```

Comment:
is there a reason for `GLUE` to be in all caps?

Reply:
my pinky likes to stay on the shift key